src/components/NotePopup/NotePopup.js

```diff
@@ -57,16 +58,37 @@ class NotePopup extends React.Component {
     this.props.openEditing();
   }
 
+  onAccept = () => {
+    core.getAnnotationManager().updateAnnotationState(this.props.annotation, 'Accepted');
```

Comment:
Can we combine these functions into one? Something like this:

```javascript
onStateUpdate = state => {
  core.getAnnotationManager().updateAnnotationState(this.props.annotation, state);  
}
```

and in the jsx below you can do something like:
```javascript
<div data-element="notePopupStateAccepted" onClick={() => this.onStateUpdate('Accept')}>{t('state.accepted')}</div>
```

The drawback of this is that this div will re-render every time when this component re-renders since we are creating a new function. But I think this is okay. Let me know if you have any concerns

Reply:
Resolved in dcb321c. Thank you!